java/maven.embedder/external/binariesembedded-list

```diff
@@ -16,43 +16,41 @@
 # under the License.
 
 C51C00206BB913CD8612B24ABD9FA98AE89719B1;commons-cli:commons-cli:1.4
-815893DF5F31DA2ECE4040FE0A12FD44B577AFAF;commons-io:commons-io:2.6
 6505A72A097D9270F7A9E7BF42C4238283247755;org.apache.commons:commons-lang3:3.8.1
 BDAAB946CA5AD20253502D873BA0C3313D141036;com.google.guava:guava:25.1-android
 FA13659F9128F4C011C8E1D06F137083B4876377;com.google.inject:guice:4.2.2:no_aop
 321C614F85F1DEA6BB08C1817C60D53B7F3552FD;org.fusesource.jansi:jansi:2.4.0
 6975DA39A7040257BD51D21A231B76C915872D38;javax.inject:javax.inject:1
 D877E195A05ACA4A2F1AD2FF14BFEC1393AF4B5E;org.slf4j:jcl-over-slf4j:1.7.36
 479C1E06DB31C432330183F5CAE684163F186146;javax.annotation:javax.annotation.api:1.2
-1637B7E8FC392E389752E79B827B883629285626;org.apache.maven:maven-artifact:3.8.6
-4D22A3FAA8880EFEF2E960BB8A00C2A0B351C46A;org.apache.maven:maven-builder-support:3.8.6
-CAF3154F8E2DBBA1CAD6625919F795F19C50EA69;org.apache.maven:maven-compat:3.8.6
-F945F1F19452214D360D453D9357275313F1CFD9;org.apache.maven:maven-core:3.8.6
-9E13CAA4D04B3DE8E990E68FB955C65E9D2E0D8F;org.apache.maven:maven-embedder:3.8.6
-8AD31867C493C63AC60A3FA42356A72D20F8457B;org.apache.maven:maven-model:3.8.6
-8323A726C657BECBFFC8EA807970935DDE3E3903;org.apache.maven:maven-model-builder:3.8.6
-4138E2E9B39F364902AC263888FEB7B2407A298E;org.apache.maven:maven-plugin-api:3.8.6
-C60A7640E1B829E60F8791B5EDF3CF56A6556ACA;org.apache.maven:maven-repository-metadata:3.8.6
+1637B7E8FC392E389752E79B827B883629285626;org.apache.maven:maven-artifact:3.8.7
+4D22A3FAA8880EFEF2E960BB8A00C2A0B351C46A;org.apache.maven:maven-builder-support:3.8.7
```

Comment:
btw I haven't updated a single hash in this file and everything is green for some reason. @ebarboni are some CI checks missing or did I forget something important?

Reply:
updated the hashes now after I noticed there is a manual `checkhash` target.

The zip is hashed and checked as regular build step since its in the `binaries-list`. I suppose the `binariesembedded-list` isn't checked because it would be redundant? I am wondering why it exists, just to list the jars?

Reply:
The file is used to get the maven coodinates for the binaries in the zip. See here:

https://github.com/apache/netbeans/blob/6afa3a9ce1cb474cbf084ab5e7f7071c4a6eca59/nbbuild/build.xml#L249-L268

That file is then further used by the build when the maven repository is generated.

Reply:
yeah I was just wondering if there is any reason to update the hashes for us. If there is: we could hook the check into the build or call it directly from the paperwork job.

If there is no reason to update the hashes since the file is only used as list of jars, we could just leave a note to not bother with it and only update artifact name and version.